src/Translation/Translator.php

```diff
@@ -167,9 +167,13 @@ public function parseKey($key)
      */
     protected function localeArray($locale)
     {
-        $locales = array_values(array_filter([$locale ?: $this->locale, $this->fallback, static::CORE_LOCALE]));
+        $locales = array_values(parent::localeArray($locale));
+
+        if (!in_array(static::CORE_LOCALE, $locales)) {
+            $locales[] = static::CORE_LOCALE;
+        }
 
-        return call_user_func($this->determineLocalesUsing ?: fn () => $locales, $locales);
+        return $locales;
```

Comment:
Does this mean that the determineLocalesUsing feature doesn't work in Winter?

Reply:
It's called in the parent class. We're just appending our core language to the end if need be.